lib/ansible/modules/cloud/openstack/os_coe_cluster.py

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/python
+
+# Copyright (c) 2018 Catalyst IT Ltd.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: os_coe_cluster
+short_description: Add/Remove COE cluster from OpenStack Cloud
+extends_documentation_fragment: openstack
+version_added: "2.7"
```

Comment:
sorry - missed this one - and I was slow. Can you change this to 2.8 and then I think we can merge this